README.md

```diff
@@ -195,6 +195,14 @@ To update dependencies:
 poetry lock
 ```
 
+### Local Dev Hokusai Version
+
+The version reported by `hokusai version` is taken from `hokusai/_version.py` file. The file is dynamically generated by [setuptools_scm](https://github.com/pypa/setuptools_scm) during CI and it is not version controlled. To have the file in local dev, run:
+
+```
+python -m setuptools_scm
+```
+
```

Comment:
if this command is not run, `hokusai version` reports v0.0.0 (the [dummy string in pyproject.toml](https://github.com/artsy/hokusai/blob/c923cc674220b45841e5c186575ff95ac1b4d28a/pyproject.toml#L6)). that causes conflict with projects that require certain versions of Hokusai (e.g. [hokusai-sandbox](https://github.com/artsy/hokusai-sandbox/blob/40d5d97e7312900ce10a035c21ac3c8d2cc38b73/hokusai/config.yml#L4)).